index.html

```diff
@@ -0,0 +1,534 @@
+      <h2 class="p-relative normal-font">The Theory of Inflation</h2>
+      <p class="p-relative">Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+      <div class="floating p-relative">
+        <img src="https://static01.nyt.com/newsgraphics/2014/03/18/theory-of-inflation/1ebaa1b972edda0f407fbaa0e010f6b50fb6792e/inflation-945.png" alt="">
+        <p class="text1">THE UNIVERSE is just under 14 billion years old. From our position in the Milky Way galaxy, we can observe a sphere that is now about 92 billion light-years across. But there's a mystery. Wherever we look, the universe has an
+          even temperature.</p>
+        <p class="text2">NOT ENOUGH TIME The universe is not old enough for light to have traveled the vast distance from one side of the universe to the other, and there has not been enough time for scattered patches of hot and cold to mix into an
+          even temperature.</p>
+        <p class="text3">DISTANT COFFEE At a smaller scale, imagine using a telescope to look a mile in one direction. You see a coffee cup, and from the amount of steam, you can estimate its temperature and how much it has cooled. </p>
+        <p class="text4">COFFEE EVERYWHERE Now turn around and look a mile in the other direction. You see a similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a similar cup in every direction, you might want to
+          look for another explanation.</p>
+        <p class="text5">STILL NOT ENOUGH TIME There has not been enough time to carry coffee cups from place to place before they get cold. But if all the coffee cups were somehow filled from a single coffee pot, all at the same time, that might
+          explain their even temperature.</p>
+        <p class="text6">INFLATION solves this problem. The theory proposes that, less than a trillionth of a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in the violently expanding energy field
+          eventually grew into the large-scale structures of the universe</p>
+        <p class="text7">FLUCTUATION Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational
+          waves predicted by Einstein.</p>
+        <p class="text8">EXPANSION Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the
+          froth and foam of the original pot.</p>
+      </div>
+      <div class="author2 p-relative">
+        <hr class="title-border">
+        <p>By LARRY BUCHANAN and JONATHAN CORUM</p>
+      </div>
+    </section>
```

Comment:
- [ ] Your theory of inflation section collapses depending on the screen size (please consider you are not the one who views this page so your page should look the same on different screen sizes), so, please set a fixed width for the text content with the background image. This will help you to have the same view on different screen sizes.

![image](https://user-images.githubusercontent.com/60184961/90852349-e32dc200-e37f-11ea-8066-70f79fd0f70d.png)